README.md

```diff
@@ -44,6 +44,11 @@ type Options struct {
 	// OnError can let users decide how to handle errors (e.g., you can suppress specific error).
 	OnError func(src, dest, string, err error) error
 
+    	// OnNameMatch returns new destination file name
+	// when source file name matches the specified pattern
+	// that has been parsed by NameRegexp
+	OnNameMatch func(re *regexp.Regexp, src string) string
```

Comment:
If the name of function is `OnNameMatch`, the output `string` is not self-descriptive.